### ISSUE TITLE
How to Run the model for Entire Indian coast?

### ISSUE THREAD

**Issue by mdsuganya1979** [kvos/CoastSat#394]
Hi,

Is it possible to run the shoreline change analysis, tidal correction and beach slope estimation for entire Indian coast? . I have already digitized shoreline for entire Indian coast. Can I use those shapefiles and import here. If so how to import?. Or should I have to split entire Indian coast into smaller ROI. Please guide me , I am unable to proceed.

Mary

**Comment by kvos** [comment]
hi @mdsuganya1979 , you need to split the indian coast in smaller ROIs and process them individually. What project are you doing this for?

**Comment by mdsuganya1979** [comment]
Thanks for the advice. I have completed National Assessment of Shoreline change for entire Indian coast from 1990-2021 using landsat, sentinel , world view and some Indian satellite images using DSAS model. My boss insisted me to publish a journal using this for entire Indian coast. I saw your model and I was interested to add tidal correction and beach slope for sandy region. Let me process for highly eroding hot spots .
